SitePackage.lua

```diff
@@ -41,18 +41,23 @@ local function logmsg(logTbl)
 end
 
 
-local function old_module_check(modT)
-    -- Check if a module is 'old'
+local function module_age(modT)
+    -- Calculate the age of a module, relative to the current toolchain generation
     -- modT should have a entry 'fn' with the module path
-    -- returns true/false
+    -- returns age (in months) between module toolchain generation and current toolchain generation
 
     local tcver = modT.fn:match("^/apps/brussel/.*/modules/(20[0-9][0-9][ab])/all/")
     if tcver == nil then return end
```

Comment:
now I see this line I'm wondering about it :p 
Should we return 0 here?

Reply:
I think better not.
if it's nil, something is wrong and it should fail.
if we return 0 we will never know something wrong.

Reply:
ah, indeed.